extensions/wikia/ImageReview/ImageReviewHelper.class.php

```diff
@@ -11,9 +11,34 @@ class ImageReviewHelper extends ImageReviewHelperBase {
 
 	private $user_id = 0;
 
+	private $image_stats_cache_keys = [];
```

Comment:
After read whole code:
Can we use separate class for mapping stats_keys into cache key names?